Doesn't compile with Zig master

### Zig Version

0.10.0-dev.3998+c25ce5bba

### Zig Language Server Version

e6238e3354e43fdecf90324e80197ab3c74b7128

### Steps to Reproduce

Attempt to build the project using the Zig version above.

### Expected Behavior

A successful compile.

### Actual Behavior

I receive the following error:
```
/home/kyle/.cache/paru/clone/zls-git/src/zls/src/Server.zig:895:9: error: pointless discard of function parameter
    _ = server;
        ^~~~~~
/home/kyle/.cache/paru/clone/zls-git/src/zls/src/Server.zig:898:16: note: used here
    return try server.hoverSymbol(writer, id, decl);
               ^~~~~~
```

--- a/src/Server.zig
+++ b/src/Server.zig
@@ -892,8 +892,6 @@ fn hoverDefinitionGlobal(server: *Server, writer: anytype, id: types.RequestId, 
     const tracy_zone = tracy.trace(@src());
     defer tracy_zone.end();
 
-    _ = server;
-
     const decl = (try server.getSymbolGlobal(pos_index, handle)) orelse return try respondGeneric(writer, id, null_result_response);
     return try server.hoverSymbol(writer, id, decl);
 }

--- a/src/analysis.zig
+++ b/src/analysis.zig
@@ -1460,8 +1460,6 @@ fn tokenRangeAppend(prev: SourceRange, token: std.zig.Token) SourceRange {
 const DocumentPosition = offsets.DocumentPosition;
 
 pub fn documentPositionContext(arena: *std.heap.ArenaAllocator, document: types.TextDocument, doc_position: DocumentPosition) !PositionContext {
-    _ = document;
-
     const line = doc_position.line;
 
     const line_mem_start = @ptrToInt(line.ptr) - @ptrToInt(document.mem.ptr);
@@ -2532,16 +2530,10 @@ fn makeInnerScope(allocator: std.mem.Allocator, context: ScopeContext, node_idx:
         if (container_field) |_| {
             if (!std.mem.eql(u8, name, "_")) {
                 var doc = if (try getDocComments(allocator, tree, decl, .Markdown)) |docs|
-                        types.MarkupContent{ .kind = .Markdown, .value = docs }
-                    else
-                        null;
-                var gop_res = try context.enums.getOrPut(allocator, .{
-                    .label = name,
-                    .kind = .Constant,
-                    .insertText = name,
-                    .insertTextFormat = .PlainText,
-                    .documentation = doc
-                });
+                    types.MarkupContent{ .kind = .Markdown, .value = docs }
+                else
+                    null;
+                var gop_res = try context.enums.getOrPut(allocator, .{ .label = name, .kind = .Constant, .insertText = name, .insertTextFormat = .PlainText, .documentation = doc });
                 if (gop_res.found_existing) {
                     if (doc) |d| allocator.free(d.value);
                 }

--- a/src/diff.zig
+++ b/src/diff.zig
@@ -33,7 +33,6 @@ pub fn edits(
     var a_trim: []const u8 = a;
     var b_trim: []const u8 = b;
     const a_trim_offset = trim_input(&a_trim, &b_trim);
-    _ = a_trim_offset;
 
     const rows = a_trim.len + 1;
     const cols = b_trim.len + 1;
@@ -239,8 +238,6 @@ pub fn get_changes(
         var range_start = group[0].pos;
         var range_len: usize = 0;
         var newText = std.ArrayList(u8).init(allocator);
-        _ = range_start;
-        _ = range_len;
         for (group) |ch| {
             switch (ch.operation) {
                 .Addition => try newText.append(ch.value.?),

